engine/admin/logging/awslogs.md

```diff
@@ -77,6 +77,115 @@ $ docker run --log-driver=awslogs \
 > **Note**:
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.
 
+### awslogs-multiline-pattern
+
+The `awslogs-multiline-pattern` specifies a regular expression pattern for identifying the start of a log message.  A log message is made of a line that matches the pattern and any following lines that don't match the pattern.  
+
+You cannot configure this option in conjunction with the `awslogs-datetime-format` option.
+
+> **Note**:
+> Multiline logging performs regular expression parsing and matching of all log messages, which may have a negative impact on logging performance.
+
+For example, to process the following log stream where new log messages start with the pattern `INFO`:
```

Comment:
For example, to process the following log stream where new log entries begin with the pattern `INFO`: